.github/workflows/foundry_test.yml

```diff
@@ -20,16 +20,16 @@ jobs:
         uses: step-security/harden-runner@91182cccc01eb5e619899d80e4e971d6181294a7 # v2.10.1
         with:
           egress-policy: audit
-
-      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           submodules: recursive
 
       - name: Use Node.js [18]
         uses: actions/setup-node@39370e3970a6d050c480ffad4ff0ed4d3fdee5af # v4.1.0
         with:
           node-version: 18
-          cache: npm
+          #cache: npm Disabling this because it causes the workflow to hang and eventually timeout
```

Comment:
Just curious why don't we remove it instead of commenting it out? Perhaps we might use it later?

Reply:
Yeah, I want to be clearly visible what was enabled before and why it's disabled. That will prevent someone adding it with good intension, but involuntarily breaking the checks.